app/src/main/res/values/strings.xml

```diff
@@ -372,6 +372,7 @@
     <string name="call_now">Call now</string>
     <string name="leave_email">Leave an email</string>
     <string name="find_locations">Find Locations</string>
+    <string name="clear_filters">CLEAR FILTERS</string>
```

Comment:
@mayank8318 don't make strings in Caps here BTW it will automatically show in caps in the dialog.